Traceback (most recent call last):
  File "train.py", line 377, in <module>
    main()
  File "train.py", line 374, in main
    train(args)
  File "train.py", line 142, in train
    best_ckpt_info = train_process(train_dataloader, val_dataloader, config, stats,save_model_dir)
  File "train.py", line 216, in train_process
    select_arm = config['select_arm']
KeyError: 'select_arm'